system/integrations/ldap/manage.rst

```diff
@@ -0,0 +1,94 @@
+Manage LDAP-Sources
+-------------------
+
+.. hint::
+
+   Please keep in mind all current limitations for the LDAP synchronization.
+   This will help to understand scopes of the following operations better.
+
+Add new source
+   Using the *New Source* button allows you to add new LDAP sources to your
+   installations. You're not limited in the number of sources, however, keep
+   in mind that many sources will also take more time to synchronize.
+
+   If you want to use LDAPs, type ``ldaps://`` instead of ``ldap://`` in front
+   of your hostname. You can use also tell Zammad to use a different port by
+   appending ``:<port number>``.
+
+   When providing a LDAPs source, Zammad will display an additional option
+   *SSL verification* that allows you to disable the verification for e.g.
+   self-signed SSL certificates.
+
+   .. tip::
+
+      Using an user filter can be a good idea if you only require a small subset
+      of your LDAP users in Zammad.
+
+      This especially affects inactive users
+         As every LDAP behaves differently on which attributes and how are set,
+         Zammad doesn't care about any flags.
+
+         Users that no longer are returned by your LDAP source will
+         automatically be set to inactive. Zammad assumes that the user was
+         deactivated.
+
+         Users will never be removed automatically!
+         If you want to remove obsolete users, use :doc:`/system/data-privacy`.
+
+      .. hint::
```

Comment:
I don't like the aesthetics of the "hint"-box within a "tip"-box. I know, it's not very specific. But maybe you can consider another visual solution here? Like a header instead?

Reply:
Okay, please have a look if you like the new approach better.

Reply:
Yes, thank you!